src/main/scala/cromiam/server/status/StatusCheckedSubsystem.scala

```diff
@@ -0,0 +1,29 @@
+package cromiam.server.status
+
+import com.softwaremill.sttp.asynchttpclient.future.AsyncHttpClientFutureBackend
+import com.softwaremill.sttp.{Uri, sttp}
+import org.broadinstitute.dsde.workbench.util.health.SubsystemStatus
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * Represents a workbench system in use by CaaS which is checked via a call to its "status" endpoint
+  */
+trait StatusCheckedSubsystem {
+  val statusUri: Uri
+
+  implicit val sttpBackend = AsyncHttpClientFutureBackend()
```

Comment:
They have other backends, e.g. one which provides `IO` but since ultimately I need a `Future` anyways to plug into `workbench-libs` it didn't seem worth it